website/docs/configuring_clients.md

```diff
@@ -4,8 +4,186 @@ title: Configuring Clients
+
+Create a new class, name it `PushApplication` and paste the following code:
+
+```java
+package com.push.pushapplication;
+
+import android.app.Application;
+
+import org.jboss.aerogear.android.unifiedpush.PushRegistrar;
+import org.jboss.aerogear.android.unifiedpush.RegistrarManager;
+import org.jboss.aerogear.android.unifiedpush.fcm.AeroGearFCMPushJsonConfiguration;
+
+public class PushApplication extends Application {
+
+    private static final String PUSH_REGISTAR_NAME = "myPushRegistar";
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        PushRegistrar pushRegistrar = RegistrarManager
+                .config(PUSH_REGISTAR_NAME, AeroGearFCMPushJsonConfiguration.class)
+                .loadConfigJson(getApplicationContext())
+                .asRegistrar();
+
+    }
+
+}
+
+```
+
+The setup of the registration happens on the `onCreate` lifecycle method called when Android first initializes your application. It will create (and store) a [PushRegistrar](https://github.com/aerogear/aerogear-android-push/blob/master/library/src/main/java/org/jboss/aerogear/android/unifiedpush/PushRegistrar.java) object based on `push-config.json` configuration declared earlier. This object together with a name (it can be anything you choose) are passed as params on the `config` factory method to create the `PushRegistrar` object.
+
+
+
+We are now ready to call *PushRegistrar:register* method to register our device in the UnifiedPush server. The following code goes after you assign the pushRegistrar in your Application class.
+
+```java
+
+    pushRegistrar.register(getApplicationContext(), new Callback<Void>() {
+        @Override
+        public void onSuccess(Void data) {
+            Log.d(TAG, "Registration Succeeded");
+            Toast.makeText(getApplicationContext(),
+                    "Yay, Device registered", Toast.LENGTH_LONG).show();
+        }
+
+        @Override
+        public void onFailure(Exception e) {
+            Log.e(TAG, e.getMessage(), e);
+            Toast.makeText(getApplicationContext(),
+                    "Ops, something is wrong :(", Toast.LENGTH_LONG).show();
+        }
+    });
+
+```
+
+Don't forget to configure the Application class in `AndroidManifest.xml`
+```xml
+<application
+    android:name=".PushApplication"
+    ...
+/>
+```
+
+That is all what is needed to register with the UnifiedPush server! Note that we didn't have to write any code to register the device with FCM. The library takes care off all the plumbing to register the device with FCM, obtain the `registrationId` and submit it to the UnifiedPush server.
+
+
+
+[NOTE]
+If you don't see the _Registration Succeeded_ popup, means that an error has occurred during the registration. Switch to the LogCat console in Android Studio to locate the exception and act accordingly.
```

Comment:
Change this to:
:::note
If you don't see the _Registration Succeeded_ popup, means that an error has occurred during the registration. Switch to the LogCat console in Android Studio to locate the exception and act accordingly.
:::